docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,256 @@
+= Synthesize a cluster
+
+:projectsyn-ig: https://vshn.chat/channel/projectsyn-ig[Project Syn IG]
+:portal-lieutenant-tenants: https://control.vshn.net/syn/lieutenanttenants/vshn-lieutenant-prod[control.vshn.net]
+:git-vshn-net: https://git.vshn.net[git.vshn.net]
+:synfra-cluster: https://rancher.vshn.net/c/c-c6j2w/monitoring[Synfra cluster].
+
+TIP: "Synthesize" is the act of making a cluster managed by Project Syn
+
+This how-to will show you how to synthesize a cluster.
+
+== Prerequisites
+
+* `cluster-admin` permissions on the cluster to synthesize.
+* Write access in the Project Syn views ("Tenants" and "Clusters") on https://control.vshn.net[control.vshn.net].
+* Access to the {synfra-cluster}.
+
+== Procedure
+
+=== Registering a tenant
+
+. Check whether there's already a Project Syn tenant for the customer whose cluster you're synthesizing on {portal-lieutenant-tenants}.
+If there's already a tenant you can directly skip to <<_registering_a_cluster,registering a cluster>>.
+
+. Ensure Lieutenant permissions on {git-vshn-net}.
+* The subgroup `cluster-catalogs` exists in the customer's group.
+* Lieutenant (GitLab account `lieutenant-prod`) has owner permissions on the subgroup `cluster-catalogs`.
+* Lieutenant has at least maintainer permissions on the customer's group to create the tenant repository.
+
+. Create tenant on https://control.vshn.net/syn/lieutenanttenants/vshn-lieutenant-prod/_create[control.vshn.net].
+To create the tenant follow the guidelines below regarding the fields to fill out.
+.. "ID": Choose a unique id for the tenant.
+This field is limited to lower-case characters `a-z`, numbers `0-9` and `-`.
+The length is limited to 1-61 characters.
++
+NOTE: This field cannot be changed after creation, it's used as the Kubernetes object name for the tenant by Lieutenant.
+
+.. "Customer": Choose the customer for which you're creating the tenant.
+.. "Odoo Customer for invoicing": Choose the Odoo customer to whom the invoices for this tenant should be sent.
+Usually this will be the same value as field "Customer".
+Check with the Service Manager for the customer which Odoo customer should be billed for the customer's Project Syn clusters.
+
+.. "Repository Type": Choose "auto".
+.. "GitRepo URL": Usually `ssh://git@git.vshn.net/<customer>/syn-tenant-repo.git`.
++
+NOTE: Make sure that the GitLab user `lieutenant-prod` has permissions to create this repository.
+
+.. "GitRepo Revision": can usually be left blank.
+.. "Global GitRepo URL": can usually be left blank.
+.. "Global GitRepo Revision": can usually be left blank.
+
+. Configure the `clusterTemplate` for the tenant.
```

Comment:
This isn't already tackled by https://github.com/projectsyn/lieutenant-operator/pull/138?

Reply:
Ok, let's leave it like that then.